tests/integration/targets/systemd_creds_decrypt/aliases

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+needs/root
+
+azp/posix/1
+skip/aix
+skip/freebsd
+skip/osx
+skip/macos
+skip/docker
```

Comment:
```suggestion
```
Same for the other test.

I was able to run the tests locally with the Debian Bookworm and the Arch Linux containers (the latter needed https://github.com/ansible-community/images/pull/109 though).

Reply:
that was just my kneejerk reaction regarding systemd stuff.

Reply:
Usually containers are the best bet for new systemd stuff, since the standard VMs in CI are RHEL and that often has older systemd.